packages/menu/menu-item.md

```diff
@@ -57,7 +57,33 @@ Content assigned to the `icon` slot will be placed at the beginning of the `<sp-
 </sp-menu>
 ```
 
-### Description slot
+##### Icon management
+
+When you use `<sp-menu-item>` elements without text content, you will need to be sure to use the `value` attribute so that the `<sp-picker>` or `<sp-menu>` element can differentiate between the available options. Further, it is important that you apply accessible labeling to the `[slot="icon"]` content as follows:
```

Comment:
```suggestion
When using `<sp-menu-item>` elements without text content, be sure to use the `value` attribute so that the `<sp-picker>` or `<sp-menu>` element can differentiate between the available options. Further, it is important to apply accessible labeling to the `[slot="icon"]` content as follows:
```

Removed 2nd person for consistency